source/_integrations/litterrobot.markdown

```diff
@@ -34,31 +34,30 @@ The Feeder-Robot is not currently supported by this integration.
-| ----------------------------- | -------- | -------------------------------------------------------------------------------- |
-| Litter Box                    | `vacuum` | Main entity that represents a Litter-Robot unit.                                 |
-| Night Light Mode              | `switch` | When turned on, automatically turns on the night light in darker settings.       |
-| Panel Lockout                 | `switch` | When turned on, disables the buttons on the unit to prevent changes to settings. |
-| Sleep Mode Start Time         | `sensor` | When sleep mode is enabled, displays the current or next sleep mode start time.  |
-| Sleep Mode End Time           | `sensor` | When sleep mode is enabled, displays the current or last sleep mode end time.    |
-| Waste Drawer                  | `sensor` | Displays the current waste drawer level.                                         |
-| Clean Cycle Wait Time Minutes | `select` | View and select the clean cycle wait time.                                       |
-| Reset Waste Drawer            | `button` | Button to reset the waste drawer level to 0%.                                    |
+| Entity                        | Domain   | Description                                                                                                                                     |
+| ----------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| Litter Box                    | `vacuum` | Main entity that represents a Litter-Robot unit.                                                                                                |
+| Night Light Mode              | `switch` | When turned on, automatically turns on the night light in darker settings.                                                                      |
+| Panel Lockout                 | `switch` | When turned on, disables the buttons on the unit to prevent changes to settings.                                                                |
+| Last Seen                     | `sensor` | Displays the time the unit was last seen / reported an update.                                                                                  |
+| Sleep Mode Start Time         | `sensor` | When sleep mode is enabled, displays the current or next sleep mode start time.                                                                 |
+| Sleep Mode End Time           | `sensor` | When sleep mode is enabled, displays the current or last sleep mode end time.                                                                   |
+| Status Code                   | `sensor` | Displays the [status code](https://github.com/natekspencer/pylitterbot/blob/884944b011f5fea9639b7d21d19fa3f7708e25a7/pylitterbot/enums.py#L44). |
```

Comment:
Linking to python code enum states seem a little too low level to link for user facing documentation.  Also the status `BONNET_REMOVED`, `BR` or ``Bonnet Removed`?

As a user there are so many codes that i'm wondering what I can even do with this. Is it just for display or would we actually make automations / alerts based on these?

Reply:
> Linking to python code enum states seem a little too low level to link for user facing documentation. Also the status `BONNET_REMOVED`, `BR` or ``Bonnet Removed`?

You're right in that it is unclear which of the 3 values is utilized as the status code.

> As a user there are so many codes that i'm wondering what I can even do with this. Is it just for display or would we actually make automations / alerts based on these?

There are a lot, and for most it is just informative. But some may require manual intervention. I already do alerting based off of some of the codes on mine. And on a different very specific one, I automatically attempt to power cycle the robot because it corrects itself after doing so. This allows for that flexibility without creating binary sensors for each variation which this other PR would have led to: https://github.com/home-assistant/core/pull/71296

Reply:
Maybe worth including an example status code? `Displays the [status code] e.g. READY, OFFLINE,   etc`

Yes, i agree that having more binary sensors for different variations is not better.